ui/common/src/main/res/values/strings.xml

```diff
@@ -569,6 +569,12 @@
   <string name="max">Max</string>
   <string name="amount">Amount</string>
   <string name="total_title">Total</string>
+  <string name="add_payment_method_button">Add payment method</string>
```

Comment:
@carlosgarridoaptoide please also add the string 'Change payment method' link to layout [here](https://www.figma.com/design/d7JmvYOmLDv3Fb6H9VTOPE/%F0%9F%93%A6-Wallet-Android---Layouts?node-id=719-6334&node-type=frame&t=I9UcxPlXRC1J8fip-0)

Reply:
Done